### PR TITLE
Add Form validation in js.

### LINKED ISSUE #35: Javascript form validation
I would like to add js form validation under lgmsoc

### PR DESCRIPTION
**Pull request by siddhi-244**
Fixes #35

### CODE REVIEW

File: form-validation/style.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (chicken-biryani) [inline form-validation/style.css:114]: Use beautifier :
CSS - https://codebeautify.org/css-beautify-minify

File: form-validation/script.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+function setSuccess(input){
+	const formControl=input.parentElement;
+	formControl.className="form-control success";
+
+}
+
+function isemail(email){
+	return /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(email);
+
+}
```
Comment (chicken-biryani) [inline form-validation/script.js:84]: Add new line at the end of file

Comment (chicken-biryani) [reply form-validation/script.js:84]: Use the link and increase readability -

JS -  https://codebeautify.org/jsviewer

File: form-validation/index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+			<i class="fas fa-exclamation-circle"></i>
+			<small>Error Message</small>
+			</div>
+			<button>Submit</button>
+		</form>
+	</div>
+	<script type="text/javascript" src="script.js"></script>
+
+</body>
+</html>
```
Comment (chicken-biryani) [inline form-validation/index.html:48]: Add a new line at the end of the file.

Comment (chicken-biryani) [reply form-validation/index.html:48]: Use the link to increase the readability.

HTML - https://codebeautify.org/htmlviewer

File: form-validation/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+- Go to the directory
+- Run the index.html file
+- Start validatibg!!!
+
+---
+
+### Screenshots
+![form](https://user-images.githubusercontent.com/69195262/124592981-b7bea380-de7b-11eb-8b60-38ec0d71c554.png)
```
Comment (chicken-biryani) [inline form-validation/README.md:30]: Use markdown beautifier -- Markdown - https://codebeautify.org/markdown-formatter

**Review by Ayushparikh-code — APPROVED** [review]
✔️👍

**Comment by siddhi-244** [comment]
Done maam @chicken-biryani

**Pull request merged**